base.pxt

```diff
@@ -45,8 +45,10 @@
   parameter "supply_duct_area_fraction", :default=>0 # fraction of conditioned floor area to set as supply duct area
   parameter "return_duct_area_fraction", :default=>0 # fraction of conditioned floor area to set as return duct area
   parameter "duct_zone", :default=>"Main Zone" # zone where supply and return ducts are located
-  parameter "duct_insulation_r_value", :default=>0 # R-value of duct insulation without air film
-  parameter "duct_air_film_r_value", :default=>1.5 # R-value of exterior air film for duct (or duct insulation)
+  parameter "duct_insulation_r_value_supply", :default=>0 # R-value of duct insulation without air film
+  parameter "duct_air_film_r_value_supply", :default=>1.7 # R-value of exterior air film for duct (or duct insulation)
```

Comment:
This parameter should be removed and always calculated in the template itself (including the logic for R=0).